### DISCUSSION TITLE
v23.0.6

### DISCUSSION THREAD

**Discussion by thaJeztah** [moby/moby#45493] [Announcements]
## 23.0.6

For a full list of pull requests and changes in this release, refer to the relevant GitHub milestones:

- [docker/cli, 23.0.6 milestone](https://github.com/docker/cli/issues?q=is%3Aclosed+milestone%3A23.0.6)
- [moby/moby, 23.0.6 milestone](https://github.com/moby/moby/issues?q=is%3Aclosed+milestone%3A23.0.6)

### Bug fixes and enhancements

- Fix vfs storage driver not working on NFS. [moby/moby#45465](https://github.com/moby/moby/pull/45465)

### Packaging Updates

- Upgrade Go to `1.19.9`. [docker/docker-ce-packaging#889](https://github.com/docker/docker-ce-packaging/pull/889),   [docker/cli#4254](https://github.com/docker/cli/pull/4254), [moby/moby#45455](https://github.com/moby/moby/pull/45455)
- Upgrade `containerd` to [v1.6.21](https://github.com/containerd/containerd/releases/tag/v1.6.21)
- Upgrade `runc` to [v1.1.7](https://github.com/opencontainers/runc/releases/tag/v1.1.7)

<hr /><em>This discussion was created from the release <a href='https://github.com/moby/moby/releases/tag/v23.0.6'>v23.0.6</a>.</em>

**Comment by polarathene** [comment]
Go 1.19.9 and Go 1.20.4 releases included backports for the `RLIMIT_NOFILE` syscall fix: https://github.com/golang/go/issues/59063

Docker [23.0.6](https://github.com/moby/moby/releases/tag/v23.0.6) (_and upcoming v24_) and containerd releases [1.6.21](https://github.com/containerd/containerd/releases/tag/v1.6.21) / [1.7.1](https://github.com/containerd/containerd/releases/tag/v1.7.1) are built with these new Go versions. `contrainerd.service` and `docker.service` can now update their configs to move away from `LimitNOFILE=infinity` :+1: 

**UPDATE (May 2025):**
- `docker.service` was updated in Docker Engine v25 release.
- `containerd.service` was updated in containerd v2.0 release.
- As of this update Docker Engine has not upgraded to containerd v2.0, thus it's not fully resolved for the user just yet.
